app/code/Magento/Directory/Setup/Patch/Data/AddDataForBulgaria.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Directory\Setup\Patch\Data;
+
+use Magento\Directory\Setup\DataInstaller;
+use Magento\Framework\App\ResourceConnection;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+use Magento\Framework\Setup\Patch\PatchVersionInterface;
+
+/**
+ * Add Bulgaria States
+ * 
+ * Class AddDataForBulgaria
+ */
+class AddDataForBulgaria implements DataPatchInterface, PatchVersionInterface
```

Comment:
Please, remove the `PatchVersionInterface` dependency. We may omit it in this particular case and remove the redundant `getVersion()` as result.